doc/release-notes-pr10740.md

```diff
@@ -0,0 +1,6 @@
+Dynamic loading of wallets
+--------------------------
+
+Previously, wallets could only be loaded at startup, by specifying `-wallet` parameters on the command line or in the bitcoin.conf file. It is now possible to load wallets dynamically at runtime by calling the `loadwallet` RPC. The wallet file/directory must be located in the `walletdir` directory in order to be loaded.
```

Comment:
> The wallet file/directory must be located in the `walletdir` directory in order to be loaded.

It works with external wallets (absolute path not related with `-walletdir`).

Reply:
Thanks promag. Release notes update. Please let me know what you think.